### DISCUSSION TITLE
Welcome to transmission-web-control Discussions!

### DISCUSSION THREAD

**Discussion by ronggang** [ronggang/transmission-web-control#509] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PrimozR** [comment]
Any chance of making this WebUI work with FreeBSD (Free/TrueNAS)?

**Comment by PrimozR** [comment]
Also, any chance of implementing Transmission 3.00 labels? Some other project already support this.

**Comment by amindavid11** [comment]
@ronggang  Hi! Just wondering why Entware has not updated to the latest version of your release https://bin.entware.net/armv7sf-k3.2/Packages.html.
What can I do to have the latest version on my router? Thanks!

**Comment by ryzhovau** [comment]
@amindavid11, it's up to OpenWrt guys, their [transmission-web-control](https://github.com/openwrt/packages/tree/master/net/transmission-web-control) is an upstream for our own package.

**Comment by hunited** [comment]
Hi, i updated hu.json.
[hu.json.txt](https://github.com/ronggang/transmission-web-control/files/7229791/hu.json.txt)

**Comment by yourvid** [comment]
<< 2022-05-30 14:53:53 >> 下载完成，正在准备安装...
<< 2022-05-30 14:53:53 >> 正在解压安装包...
<< 2022-05-30 14:53:53 >> 正在复制安装包...
**cp: cannot stat '/tmp/tr-web-control/transmission-web-control/src/.': No such file or directory**
<< 2022-05-30 14:53:53 >> 正在设置权限，大约需要一分钟 ...
<< 2022-05-30 14:53:53 >> Transmission Web Control 安装完成!
<< 2022-05-30 14:53:53 >> 正在清理安装包...
<< 2022-05-30 14:53:53 >> 安装脚本执行完成。如遇到问题请查看：https://github.com/ronggang/transmission-web-control/wiki 
<< 2022-05-30 14:53:53 >> == 结束 ==

群晖自定义脚本运行报错：/tmp/tr-web-control/transmission-web-control/src/ 目录不存在
